config.mk

```diff
@@ -13,6 +13,7 @@ MAN = $(LPREFIX)/man
 DOC = $(SPREFIX)/doc
 LIC = $(SPREFIX)/licenses
 LGUSER = $(shell users | awk '{ print $$1 }')
```

Comment:
use id command for effective user as well.
```shell
LGUSER = $(shell id -un)
```

Reply:
This will cause files to be written for root:root